CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## Version 1.17.1
+* Release date: April 29, 2020
+* Release status: General Availability
+* Hotfix for https://github.com/microsoft/azuredatastudio/milestone/54?closed=1
```

Comment:
I'm guessing this is standard on how we do it? I was thinking of linking back to the specific issue.

Reply:
This is how vs code does it too - makes sense to me (that way we don't have to think about whether there's going to be multiple issues or not. We just always do the same thing)

Reply:
Yeah, in this case linking to the issue would make sense, but I tend to agree with @Charles-Gagnon  here to just consistently link to the milestone so we can cut\paste these hotfix notes.